src/plots/cartesian/axes.js

```diff
@@ -1792,7 +1795,9 @@ axes.doTicks = function(gd, axid, skipTitle) {
         }
         else {
             flipit = (axside === 'right') ? 1 : -1;
-            labely = function(d) { return d.dy + d.fontSize / 2 - labelShift * flipit; };
+            labely = function(d) {
+                return d.dy + d.fontSize * 0.35 - labelShift * flipit;
```

Comment:
hah, I found the 0.35 value empirically, but polar had the same one, and it turns out there's a rational basis for exactly this value. https://github.com/plotly/plotly.js/pull/1854/commits/16bae9ca0be3328e6b0f6b7d975ffdb0eafd6720